index.html

```diff
@@ -34,6 +34,155 @@ <h1>Charlie<br>Giorgania</h1>
   <img src="svg/image_geometry_2.svg" class="blockImage" alt="">
 </div>
 
+<section>
+  <header>
+    <h1>Projects</h1>   
+  </header>
+  <div>
+    <figure> 
+      <img src="Images/project-image-1.jpg" class="blockImage" alt="">
+    </figure>
+    <div class="grid-container">
+      <p>Project name goes here</p>
+      <div class="code-buttons">
+        <ul>
+          <li><button>HTML/CSS</button></li>
+          <li><button>Ruby on Rails</button></li>
+          <li><button>Javascript</button></li>
+        </ul>   
+      </div>
+      <button class="arrow-click">See this project <img src="Images/Arrow.png" alt=""></button>
```

Comment:
- [ ] For each button, add the required button styling based on the design: Active, Hover and Pressed